tests/cpp_api/audio_tests/audio_tests.py

```diff
@@ -25,7 +25,8 @@
 import argparse
 
 test_case_augmentation_map = {
-    0: "audio_decoder"
+    0: "audio_decoder",
```

Comment:
please take the test cases as list of strings instead of using a map

Reply:
Test case argument is passed as int in audio_tests.cpp so we use the keys for passing the case list and values for printing the augmentation names during results.